erpnext/patches/v10_0/add_default_cash_flow_mappers.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+
+def execute():
+    mappers = [
```

Comment:
These default mappers will also be initiated on fresh install.

Reply:
I forgot. Thanks